### DISCUSSION TITLE
403 Forbidden Error

### DISCUSSION THREAD

**Discussion by pietrobanana** [docksal/docksal#1613] [Q&A]
I am new to using Docksal, but when I try to start using "fin start" I get my Wordpress site pulled up but when i look in the browser it says "403 Forbidden You don't have permission to access / on this server."

I'm trying to spin up my existing wp project using Docksal. 

This is in my docksal.env `DOCKSAL_STACK=default` 
 
What am I missing in order to connect to my db and site? 

Thanks,
Jorge

**Comment by lmakarov** [comment]
Make sure your website is in the `docroot` directory (default in Docksal) inside of the project folder.

You can also override the docroot:

```
fin config set DOCROOT=<directory>
fin project start
```

If you have the website in the root of the project directory, then

```
fin config set DOCROOT=.
fin project start
```

See https://docs.docksal.io/stack/configuration-variables/
